### ISSUE TITLE
[Issue]: dpkg configuration error on Debian due to Systemd

### ISSUE THREAD

**Issue by nairboon** [RadeonOpenCompute/rocm_smi_lib#184]
`rocm-smi-lib` can't be installed on Debian testing, because dpkg-configure fails for `rocm-smi-lib (7.2.0.60102-119~22.04)`:

```
Removed '/etc/systemd/system/timers.target.wants/logrotate.timer'.
Created symlink '/etc/systemd/system/timers.target.wants/logrotate.timer' → '/usr/lib/systemd/system/logrotate.timer'.
--now can only be used with verb enable, disable, or mask
```
which then fails in https://github.com/ROCm/rocm_smi_lib/blob/323ab1105dcee578454e1feff0b01bb3a98fa01f/DEBIAN/postinst.in#L110 

this line might need to be `exit 0` because it fails for me with
`return: can only 'return' from a function or sourced script`

The actual problem is a change of behaviour in systemd, the above code should have worked from systemd v252+ but in v256 the option `--now` fails for the `reenable` command in https://github.com/ROCm/rocm_smi_lib/blob/323ab1105dcee578454e1feff0b01bb3a98fa01f/DEBIAN/postinst.in#L83 

see https://github.com/systemd/systemd/issues/31541 

Version: rocm 6.1.2

**Comment by dmitrii-galantsev** [comment]
You're absolutely right @nairboon return 0 makes no sense there.

**Comment by dmitrii-galantsev** [comment]
I submitted a patch internally it should make its way up to github in the next couple days

**Comment by ppanchad-amd** [comment]
@nairboon Fixed in the latest ROCm 6.2. Closing ticket. Thanks
